### DISCUSSION TITLE
[TW3.2.4] bg-color only renders when applying /opacity value

### DISCUSSION THREAD

**Discussion by hphoeksma** [tailwindlabs/tailwindcss#10451] [Help] [answered]
Hi, I am experiencing an issue where classes like `bg-red-500` are only rendering correct css output when writing it out fully with the opacity setting like `bg-red-500/100`.
My config file is unmodified and I run it through postCSS without additional plugins other than auto-prefixer.

Anyone the same issue? Anyone an idea how to make this work normally?

**Comment by wongjn** [comment]
Please consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by hphoeksma** [reply]
I am not really sure how to make that happen as it is a private package for a client ;-)

**Reply by wongjn** [reply]
You could look at creating a copy of the git repository to the bare minimum code that reproduces the issue. This could also serve as troubleshooting and could help isolate the cause.

**Reply by hphoeksma** [reply]
@wongjn true - will see if I can get that done, thanks for your response.

**Comment by hphoeksma** [comment, marked as answer]
Figured out what went wrong. I loaded the generated styles inside a style tag (as in critical styles). But custom properties won't work inside that notation thus killing all the output using cp's 🐵
